Zilon.Core/Zilon.Bot.Players/Logics/MoveLogicStateBase.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+using Zilon.Core.Tactics;
+using Zilon.Core.Tactics.Behaviour;
+using Zilon.Core.Tactics.Behaviour.Bots;
+using Zilon.Core.Tactics.Spatial;
+
+namespace Zilon.Bot.Players.Logics
+{
+    public abstract class MoveLogicStateBase: LogicStateBase
+    {
+        protected MoveTask MoveTask { get; set; }
+        protected IdleTask IdleTask { get; set; }
+
+        protected IDecisionSource DecisionSource { get; }
+
+        protected ISectorMap Map { get; }
+
+        public MoveLogicStateBase(IDecisionSource decisionSource, ISectorManager sectorManager)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of this constructor to 'protected'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5331978)